stablehlo/reference/Ops.cpp

```diff
@@ -224,6 +225,47 @@ Tensor evalTransposeOp(const Tensor &operand, ArrayRef<int64_t> permutation,
   return result;
 }
 
+// ***********************************************************************
+// This is an simplified implementation of while op semantics. The
+// simplification is based on iterating the loop `dummy_limit` number of times
+// if the loop condition is `true`. The simplifiction is done as compare op is
+// not supported. To be corrected and improved as part of #967 and #992
+// respectively.
+// ***********************************************************************
+SmallVector<Tensor> evalWhileOp(ArrayRef<Tensor> runtimeInputs, Region &cond,
```

Comment:
Do we have to say "runtime" here and below? In nearby functions, we simply say stuff like "lhs" and "result".

Reply:
Apologies if this conversation has already happened - is there any reason to not sync these with (either ODS or spec) names -`operands` in this case?

Reply:
That is a good point. The idea was very much  to keep them in sync with ODS. But I admit that it is not documented anywhere.  Point noted.

Reply:
Even if this is not documented, let's follow the style of the nearby code which doesn't use "runtime" prefixes in `evalFooOp` functions?